sample/EventHubTrigger/index.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (context, input) {
         var bindingData = context.bindingData,
             eventProperties = bindingData.propertiesArray[i],
             systemProperties = bindingData.systemPropertiesArray[i],
-            id = input[i].value;
+            id = JSON.parse(input[i]).value;
```

Comment:
v1 didn't have to do this - if the value is JSON I think we should handle this automatically, as we did before